.github/workflows/build_db.yaml

```
@@ -0,0 +1,75 @@
+name: Create & Update DB
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    defaults:
+        run:
+          shell: bash -el {0}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Build environment
+      uses: ./.github/actions/build-conda-env
+      with:
+        env_file: environment.yaml
+
+    - name: Activate environment & check conda info
+      run: |
+        conda activate $(grep name environment.yaml | awk '{ print $2}')
+        conda info
+        echo "Install cfdb"
+        pip install -e .
+
+    - name: Retrieve previous DB table from artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: cfdb
+        path: /home/runner/work/czi-conda-forge-db/cf-database.db
```

```suggestion
        path: ${{ github.workspace }}/../cf-database.db
```

This is equivalent but why on the directory above?

---

Thanks, this could be changed if needed, though, then we would need to pass the path to the new location into the execution command as, by default, it will look into the dir that called the command for the DB file